op-challenger/config/config_test.go

```diff
@@ -43,37 +45,72 @@ var cannonTraceTypes = []types.TraceType{types.TraceTypeCannon, types.TraceTypeP
 var asteriscTraceTypes = []types.TraceType{types.TraceTypeAsterisc}
 var asteriscKonaTraceTypes = []types.TraceType{types.TraceTypeAsteriscKona}
 
-func applyValidConfigForCannon(cfg *Config) {
-	cfg.Cannon.VmBin = validCannonBin
-	cfg.Cannon.Server = validCannonOpProgramBin
+func ensureExists(path string) error {
+	_, err := os.Stat(path)
+	if err == nil {
+		return nil
+	}
+	err = os.MkdirAll(filepath.Dir(path), os.ModePerm)
+	if err != nil {
+		return err
+	}
+	file, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	return file.Close()
+}
+
+func applyValidConfigForCannon(t *testing.T, cfg *Config) {
+
+	vmBin := filepath.Join(t.TempDir(), validCannonBin)
+	server := filepath.Join(t.TempDir(), validCannonOpProgramBin)
```

Comment:
(nit) You can just create a single tmpdir and reuse it.

Reply:
Done in 0dedea3c5e541f9005d86acce25a163868321574 .